src/components/UserPage/CommentGrid.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ul :class="$style.list">
+    <li v-for="{ text, item } in comments" :key="item.id">
+      <comment :text="text" :item="item" />
+    </li>
+  </ul>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue'
+import { ItemSummary } from '/@/lib/apis'
+import Comment from './Comment.vue'
+
+export default defineComponent({
+  name: 'CommentGrid',
+  components: {
+    Comment
+  },
+  props: {
+    comments: {
+      type: Array as PropType<{ text: string; item: ItemSummary }[]>,
```

Comment:
```suggestion
import { Comment as CommentType } from '/@/lib/apis'
import Comment from './Comment.vue'

export default defineComponent({
  name: 'CommentGrid',
  components: {
    Comment
  },
  props: {
    comments: {
      type: Array as PropType<CommentType[]>,
```
型が存在してるのでこれを使ったほうがいいです 🖖